src/main/webapp/scripts/sessioncache.js

```diff
@@ -33,60 +36,49 @@ class SessionCache {
      */
     this.sessionInformation_ = null;
```

Comment:
Can we change all this sessionInformation stuff to just session? So getSession() instead of getSessionInformation() , etc